cmd/podman/libpodruntime/runtime.go

```diff
@@ -1,17 +1,59 @@
 package libpodruntime
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
+
 	"github.com/containers/storage"
 	"github.com/projectatomic/libpod/libpod"
 	"github.com/urfave/cli"
 )
 
 // GetRuntime generates a new libpod runtime configured by command line options
 func GetRuntime(c *cli.Context) (*libpod.Runtime, error) {
-	storageOpts := storage.DefaultStoreOptions
+	storageOpts, err := GetDefaultStoreOptions()
+	if err != nil {
+		return nil, err
+	}
 	return GetRuntimeWithStorageOpts(c, &storageOpts)
 }
 
+func GetRootlessStorageOpts() (storage.StoreOptions, error) {
+	var opts storage.StoreOptions
+
+	home := os.Getenv("HOME")
+	runtimePath := os.Getenv("XDG_RUNTIME_DIR")
+	if runtimePath == "" {
+		runtimePath = filepath.Join(home, "rundir")
+	}
+	if home == "" {
```

Comment:
Can we move is up and place it right after `home := os.Getenv("HOME")` call?

[1]: https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.7.html#variables

Reply:
I think the opposite might be better (move the `os.Getenv("HOME")` down here)

Reply:
moved up.  I cannot move the `os.Getenv` down here as it is used in the `XDG_RUNTIME_DIR` fallback